<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookOrder</class>
 <widget class="QMainWindow" name="BookOrder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>761</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>860</width>
    <height>761</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>860</width>
    <height>761</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Take a book - Cataloging Department</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Эмблема2.png</normaloff>Эмблема2.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_pic">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>861</width>
      <height>761</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>order2.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="lab_user_name">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 55, 57);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_back">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>60</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(229, 228, 224);
color: rgb(84, 55, 59);</string>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QTableView" name="tab_book">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>170</y>
      <width>601</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lab_nazv">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(252, 251, 248);
color: rgb(85, 54, 59);</string>
    </property>
    <property name="text">
     <string>Взять книгу</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lab_book">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(252, 251, 248);
color: rgb(85, 54, 59);</string>
    </property>
    <property name="text">
     <string>    Название книги:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_numbook">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(252, 251, 248);
color: rgb(85, 54, 59);</string>
    </property>
    <property name="text">
     <string>   Количество книг</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_take">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Small Semibold</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(229, 228, 224);
color: rgb(85, 53, 56);</string>
    </property>
    <property name="text">
     <string>Взять</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lab_info">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>741</y>
      <width>861</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(156, 128, 124);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string> По всем вопросам - обращайтесь к библиотекарю.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="cB_book">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lab_naz_r">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(252, 251, 248);
color: rgb(85, 54, 59);</string>
    </property>
    <property name="text">
     <string>   Читатель:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_naz_r">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(86, 54, 59);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_book">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(252, 251, 248);
color: rgb(85, 54, 59);</string>
    </property>
    <property name="text">
     <string>Книги</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_order">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>120</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(252, 251, 248);
color: rgb(85, 54, 59);</string>
    </property>
    <property name="text">
     <string>Абонементы</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lab_nazv_ext">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>420</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(252, 251, 248);
color: rgb(85, 54, 59);</string>
    </property>
    <property name="text">
     <string>Продлить</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lab_book_ext">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(252, 251, 248);
color: rgb(85, 54, 59);</string>
    </property>
    <property name="text">
     <string>    Название книги:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cB_book_ext">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>480</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ext">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>580</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Small Semibold</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(229, 228, 224);
color: rgb(85, 53, 56);</string>
    </property>
    <property name="text">
     <string>Продлить</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="te_numb">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_read_ex">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>540</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(86, 54, 59);</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_naz_r_ext">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(252, 251, 248);
color: rgb(85, 54, 59);</string>
    </property>
    <property name="text">
     <string>   Читатель:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
